packages/katello/katello/katello.spec

```diff
@@ -167,10 +167,8 @@ Useful utilities for debug info collecting
 %package -n foreman-proxy-content
 Summary: Provides a federation of katello services
 BuildArch: noarch
-Requires: findutils
```

Comment:
This is still used: https://github.com/theforeman/foreman-packaging/blob/0d038ed8b626b0e8dc53690c59cbe6187c473c6a/packages/katello/katello/foreman-proxy-content.cron#L5

Reply:
good catch, reverted that part.